codec-http/src/main/java/io/netty/handler/codec/spdy/SpdyCodecUtil.java

```diff
@@ -305,9 +305,6 @@ static void validateHeaderName(CharSequence name) {
                 throw new IllegalArgumentException(
                         "name contains null character: " + name);
             }
-            if (c >= 'A' && c <= 'Z') {
```

Comment:
@buchgr - The fix needs to go back to 4.0. These header changes will not go back to 4.0.